doc/conf.py

```diff
@@ -58,10 +58,7 @@
 
 # Add any URL patterns to ignore (e.g. for private sites, or sites that
 # are frequently down).
-linkcheck_ignore = [
-    r"^https://jira.lsstcorp.org/browse/",
-    r"^https://dev.lsstcorp.org/trac"
-]
+linkcheck_ignore = [r"^https://jira.lsstcorp.org/browse/", r"^https://dev.lsstcorp.org/trac"]
```

Comment:
We should not have any links to Trac at this point?

Reply:
:+1: